client/src/components/imgdetail/Imgdetail.jsx

```diff
@@ -0,0 +1,27 @@
+import styles from "./ImgDetail.module.css";
+
+function ImgDetails() {
+  return (
+    <div className={styles.container}>
+      <img
+        className={styles.img}
+        src="https://media.istockphoto.com/id/481229372/fr/photo/galaxie-spirale-illustration-de-la-voie-lact%C3%A9e.jpg?s=612x612&w=0&k=20&c=b2PsgdKm7IwiJfrGbhq2SgNbPXrgczuE3QPQ712g5Z4="
+        alt="test"
```

Comment:
"test", is that supposed to be a well formed alt ?

Reply:
"Test" was just for test matters, it's not the final Alt. But I'll update it during the next push since the images will be fetched from the API and the Alt too.